test/functional/eval/system_spec.lua

```diff
@@ -89,7 +89,9 @@ describe('system()', function()
     end)
 
     it('does NOT run in shell', function()
-      if not iswin() then
+      if iswin() then
+        eq("C:\\Windows\\System32\\cmd.exe\n", eval("system(['where', 'cmd'])"))
```

Comment:
This isn't testing anything useful

Reply:
I can remove this and disable the test on Windows if that's better.
I assumed that this test should not use `&shell`

Reply:
Yes but the test is supposed to check that env vars aren't expanded